### DISCUSSION TITLE
Discord, Joining, and Formatting

### DISCUSSION THREAD

**Discussion by bones-01** [OffprintStudios/dragonfish-old#840] [Q&A] [answered]
Hi, I learned about offprint through an author who prefers to have their work off of fimfiction, but I'm having difficulty really doing all that much on the site.  I can't join the site by registering, I've heard something about a discord but there's no details on that anywhere, and I have little idea of how to contact anyone involved with the site without an account I can't make.  I can try to read their work without any of that, but the formatting button doesn't work and I was trying to figure out if that's a bug or an incomplete feature.  I much prefer reading in Times New Roman, because that's what I grew up with.

**Comment by Figments** [comment, marked as answer]
hey there! sorry for not making it obvious where the Discord invite link is—we had one on the front page for a while but it got ignored when we released the All Jokes Aside update a week ago. we'll be putting a link back up on the home page very soon, but in the meantime you can use [this link](https://discord.gg/qZRDSY2W) to join it! just know you'll have to agree to our rules before participating, but it's effector the same as agreeing to the site's Constitution, which is required for an account regardless. 

from there, you can head over to the suggestion box channel to talk with the devs directly. we hope to see you there soon!

**Comment by SigmasonicX** [comment]
Regarding the formatting button, that is indeed not implemented yet.
About the project in general, we understand that we haven't been as transparent as we could have been about the site's current state and development timeline. The site is in a Closed Alpha state, so we're intentionally restricting the number of new users until we're ready to open up fully. That said, it was our mistake in neglecting the Discord link with our latest update.
